greenboot-script-runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+run_required_scripts () {
+  required_greenboot_scripts=`ls /etc/greenboot.d/required/*.sh`
```

Comment:
what's the most robust way to gather all scripts in this dir and handle the empty-dir case?

Reply:
this looks reasonable to me

Reply:
I usually use `find` exactly because of the empty dir case. `ls` will error out.
E.g. something like: `find /etc/greenboot.d/required -name '*.sh'`